README.md

```diff
@@ -14,8 +44,54 @@ To combine lists of events in to a calendar:
 python yaml2ics.py example/test_calendar.yaml example/another_calendar.yaml
 ```
 
+## Syntax
+
+Please see `example/test_calendar.yaml` for a full demo including
+explanations.  Below is a minimal template that shows the basic idea:
+
+```yaml
+name: Calendar Name
+timezone: Europe/Helsinki  # default timezoene for events, optional
+
+events:
+  - summary: The event title
+    begin: 2021-09-21 15:00:00
+    duration:
+	  minutes: 30
+    location: |
+      https://meet.jit.si/example
+    description: |
+	  In this meeting we will ...
+```
+
+## Contributing
+
+Contributions are welcomed! This project is still in active development
+and should be considered beta.
+
+[flit](https://flit.readthedocs.io/en/latest/) is used to manage the project.
+
+```
+pip install flit
+pip install -r requirements.txt  # temporary workaround for ics-py
+flit install -s  # make an editable/inplace installation
+```
+
 To test:
 
 ```
-PYTHONPATH=. pytest
+pytest
 ```
+
+[black](https://github.com/psf/black) and other linters are used to auto-format
+files (and enforced by CI). To install the git hooks, use `pre-commit install`.
+To run the tests/auto-formatting manually, use `pre-commit run
+--all-files`.
+
+Releases are automatically pushed on PyPi by the CI when pushing a tag
+following `*.*`.
+
+## See also
+
+* https://github.com/priyeshpatel/yaml-to-ical - older (~2014) idea of
+  the same thing
```

Comment:
Not sure this is relevant; that package was updated 8 years ago, and we didn't look at it while developing.

Reply:
ok removing